tests/php/InlineCssTest.php

```
@@ -545,6 +647,193 @@ static function ( $handle ) use ( $paths ) {
+					$this->assertSame(
+						$expected,
+						$this->invoke_private( $cache, 'core_will_inline', array( 'foo' ) ),
+						sprintf( 'Unexpected inline decision at %d bytes on WP %s', $bytes, $version )
+					);
+				}
+			}
+		} finally {
+			$this->cleanup_temp_files( $files );
+			unset( $GLOBALS['wp_version'] );
+		}
+	}
```

PHPMD (2.15.0)

[error] test_core_will_inline_budget_boundary_is_version_aware accesses the super-global variable $GLOBALS. (undefined)

(Superglobals)